CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Version 0.5.0-dev  (changes since 0.5.0 go here)
 
 ### Features
+* `DissimilarityMatrix` now has a new constructor method called `from_iterable`. ([#1343](https://github.com/biocore/scikit-bio/issues/1343)).
```

Comment:
No problem. Did I do the `validate` kwarg backwards? Right now `True` provides the previous behavior (default).